scripts/container-only/wkdev-setup-default-clang

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+# Copyright 2024 Igalia S.L.
+# SPDX-License: MIT
+
+if [ -f "${WKDEV_SDK}/.wkdev-sdk-root" ]; then
+    source "${WKDEV_SDK}/utilities/application.sh"
+else
+    echo "Please set \${WKDEV_SDK} to point to the root of the wkdev-sdk checkout."
+    exit 1
+fi
+
+init_application "${0}" "Installs and creates symlinks to set default clang executables" container-only
+
+argsparse_use_option "=version:" "The clang version" "mandatory" "type:uint"
+
+
+argsparse_usage_description="$(cat <<EOF
+<< Purpose >>
+
+    Provides an easy way to install and switch between clang toolchains.
+
+<< Examples >>
+
+    $ ${application_name} --version=17
+EOF
+)"
+
+run() {
+
+    argsparse_parse_options "${@}"
+    local version="${program_options["version"]}"
+
+    echo ""
+
+    # Sanity check versions Ubuntu actually has.
+    if (( $version < 14 )) || (( $version > 18)); then
```

Comment:
I'm a fan of `${foo}` over `$foo` -- can you adapt to keep things consistent wrt to the other scripts?